net/qpid-proton/Portfile

```diff
@@ -58,3 +63,16 @@ variant swig description {With built-in support for SWIG so the bindings can be
                             port:swig-go \
                             port:swig-ruby
 }
+
+post-build {
+    set scriptname   runtests.sh
+
+    set script [open "${worksrcpath}/../build/${scriptname}" w 0644]
+    puts ${script} "#!/bin/bash"
+    puts ${script} "source ./config.sh"
+    puts ${script} "make test"
+    close ${script}
+    file attributes ${worksrcpath}/../build/${scriptname} -owner macports -group admin
```

Comment:
There shouldn't be a need to modify the file's ownership. And there's no guarantee that the unprivileged user's name is "macports", nor that MacPorts was even configured to use an unprivileged user.